src/transformation/utils/scope.ts

```diff
@@ -77,12 +77,14 @@ export function findScope(context: TransformationContext, scopeTypes: ScopeType)
 }
 
 const scopeIdCounters = new WeakMap<TransformationContext, number>();
-export function pushScope(context: TransformationContext, scopeType: ScopeType): void {
+export function pushScope(context: TransformationContext, scopeType: ScopeType): Scope {
```

Comment:
There should a bunch of usages of `peekScope` right after `pushScope`, would be nice to simplify them now

Reply:
Only found one in switch